src/v3/src/components/ConsentHeader/ConsentHeader.tsx

```diff
@@ -91,6 +91,7 @@ const ConsentHeader: FunctionComponent = () => {
               <Link
                 href={href}
                 target="_blank"
+                rel="noopener noreferrer"
```

Comment:
added these attributes to any links that have `target="_blank"` for security.  This will prevent possible redirects to a malicious URL.
Reference: https://mui.com/material-ui/react-link/#security